src/templates/footer_fragment.handlebars

```diff
@@ -0,0 +1,120 @@
+<div>
+  <style>
+    .mw-body {
+      padding-bottom: 0;
+    }
+
+    .pre-footer-wrapper,
+    .footer-wrapper,
+    .post-footer-wrapper {
+      width: 55.8em;
+      margin: 0px auto;
+      font-family: 'Linux Libertine', Georgia, Times, serif;
+    }
+
+    .pre-footer-wrapper div {
+      margin-bottom: 10px;
+    }
+
+    .footer-wrapper {
+      background-color: white;
+      border: 2px solid lightgrey;
+      border-radius: 10px;
+      margin-bottom: 25px;
+    }
+
+    .footer-wrapper a {
+      color: #0084b4;
+      text-decoration: none;
+      font-weight: bold;
+      letter-spacing: 1px;
+    }
+
+    .footer-wrapper a:hover {
+      color: #0084b4;
+      /* text-decoration: ; */
+    }
+
+    .footer-logo,
+    .footer-titles,
+    .footer-sharing {
+      position: relative;
+      float: left;
+      width: 200px;
+      text-align: center;
+    }
+
+    .footer-logo {
+      width: 100px;
+      padding-left: 10px;
+    }
+
+    .footer-titles,
+    .footer-sharing {
+      padding-top: 20px;
+    }
+
+    .footer-titles {
+      width: 570px;
+    }
+
+    .footer-titles-title {
+      font-size: 19px;
+      margin-bottom: 10px;
+      font-weight: bolder;
+    }
+
+    .footer-sharing-icons {
+      width: 168px;
+      margin: 0 auto;
+    }
+
+    .footer-sharing-icon {
+      width: 32px;
+      height: 32px;
+      float: left;
+      margin: 5px;
+    }
+  </style>
+  <div class="pre-footer-wrapper">
```

Comment:
Can we move this inside of the footer?

## Instead of

It looks bit awkward and unclear which labels come from us, and what was the original ZIM:

> ![before](https://user-images.githubusercontent.com/157609/76228057-82228f80-6220-11ea-9885-6a456a4259bb.png)

## Do this

If we put everything within the same "bagde", it will be much easier to understand:

> ![footer](https://user-images.githubusercontent.com/157609/76228060-8353bc80-6220-11ea-8eb9-e938253fcaf4.png)

Thanks!